src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -125,6 +125,19 @@ public SendLetterController(
         return noContent().build();
     }
 
+    @PostMapping(path = "/check-print-state")
```

Comment:
can we move it to a different controller may be `StatusCheckerController`? what are your thoughts?